eudevfin/reports-ui/src/main/java/org/devgateway/eudevfin/reports/ui/pages/ReportsPage.java

```diff
@@ -160,7 +160,7 @@ private void addNetODATable () {
                         if (resultSet.get(i).get(NUMBER_OF_YEARS - 1) == null) {
                             resultSet.get(i).add(null);
                         } else {
-                            if (resultSet.get(i).get(NUMBER_OF_YEARS) == null) {
+                            if (resultSet.get(i).size()<=NUMBER_OF_YEARS || resultSet.get(i).get(NUMBER_OF_YEARS) == null) {
```

Comment:
i don't like this fix, need to dig a bit more into why i dont get all the items here...